FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ManagedFeedImage+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Andreas Link on 13.06.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+internal class ManagedFeedImage: NSManagedObject {
+	@NSManaged internal var id: UUID
+	@NSManaged internal var imageDescription: String?
+	@NSManaged internal var location: String?
+	@NSManaged internal var url: URL
+	@NSManaged internal var cache: ManagedCache
```

Comment:
As per my previous comment, the `internal` keyword should be omitted.
```suggestion
	@NSManaged var id: UUID
	@NSManaged var imageDescription: String?
	@NSManaged var location: String?
	@NSManaged var url: URL
	@NSManaged var cache: ManagedCache
```